cocos/dragon-bones/ArmatureSystem.ts

```diff
@@ -0,0 +1,65 @@
+import { director } from '../core/director';
+import { System } from '../core/components';
+import { ArmatureDisplay } from './ArmatureDisplay';
+import { legacyCC } from '../core/global-exports';
+import { errorID } from '../core';
+
+export class ArmatureSystem extends System {
+    /**
+     * @en
+     * The ID flag of the system.
+     * @zh
+     * 此系统的 ID 标记。
+     */
+    static readonly ID = 'ARMATURE';
+
+    private static instance: ArmatureSystem;
+
+    private constructor () {
+        super();
+        if (ArmatureSystem.instance) {
+            errorID(12101);
+        }
```

Comment:
Just found another issue, if use new multiple times, it won't trigger any issue, 

```
let singleton1 = new ArmatureSystem();
let singleton2 = new ArmatureSystem();
let singleton3 = new ArmatureSystem();
// Still work
```

we should assign instance here instead of in `getInstance`.